server.js

```diff
@@ -1,11 +1,15 @@
 const express = require('express');
-const graphqlHTTP = require('express-graphql');
-const app = express();
+const { graphqlHTTP } = require('express-graphql');
 const schema = require('./schema');
-app.use('/graphql', graphqlHTTP({
-  schema: schema,
-  graphiql: true
-}));
 
-var PORT = process.env.PORT || 5000;
+const app = express();
+
+app.use('/graphql', 
+  graphqlHTTP({
+    schema: schema,
+    graphiql: true
+  }),
+);
+
+const PORT = process.env.PORT || 5000;
```

Comment:
You can turn this from a var to const because it is not being reassigned. Also u should never use var, instead if ur variable is being reassigned, use (let)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let]